src/dynamodb_encryption_sdk/structures.py

```diff
@@ -378,10 +378,10 @@ def refresh_indexed_attributes(self, client):
         table = client.describe_table(TableName=self.name)['Table']
         self._primary_index = TableIndex.from_key_schema(table['KeySchema'])
 
-        self._secondary_indexes = set()
+        self._secondary_indexes = []
```

Comment:
Is there anything that relied on the unordered property of `set` / anything that will be surprised by order and/or (error-case) duplication being introduced?

I didn't see anything in my quick look but you know the code base best. :)

Reply:
No, we're actually not doing anything with it at all at the moment, it's just being opportunistically collected since it's all in the same API call. I went with the change to list rather than tuple to retain the mutability and the change away from set rather than adding hash capability to TableIndex because I think the later is something that, while it might be useful later, needs more thought if we want to do.